src/blocks/features/Features/Features.js

```diff
@@ -52,6 +53,7 @@ const Features = ({
   icon_height = 60,
   center = false,
   cta_button_text = '',
+  background_color = ""
```

Comment:
you can use default value here for background_color , also please use camelCase if possible :)